app/components/Resources/ResourcesTable.js

```diff
@@ -98,16 +101,16 @@ class ResourcesTable extends Component {
 		let page = 0;
 		if(this.state.openFilter) {
 			this.setState({
-				page: page,
+				page,
 				resources: this.state.allResources,
-				currentPage: this.state.allResources.slice(page, page + 9),
+				currentPage: this.state.allResources.slice(page, resultsPerPage),
```

Comment:
Yeah, it's a minor thing. My only qualm is that the expression looks like it happily takes a variable, `page`, but if that variable actually varied, the expression wouldn't work. We can leave it.